build_projects.yml

```diff
@@ -1,8 +1,10 @@
 trigger:
 - master
 - dev/*
+- staging/*
```

Comment:
Why staging if there is dev for triggering the build?

Reply:
Alignment with: https://github.com/analogdevicesinc/no-OS/commit/07f6e2bc73a5f49e027438d5b91b631853da4ffb
We either remove `staging` from both files or keep them.
If we want custom branch that builds only projects, then a more suggestive prefix should be added.

Reply:
Alignment with: 07f6e2b
We either remove staging from both files or keep them.
If we want custom branch that builds only projects, then a more suggestive prefix should be added.

Reply:
I would replace dev then instead of adding staging and add main here too

Reply:
Yes, will do so, since linux repo uses same approach (only `staging/*`, no `dev/*`)